docs/SQL/extensibility/dependencies.md

```diff
@@ -0,0 +1,107 @@
+---
+sidebar_position: 1
+title: Dependencies
+id: dependencies
+description: Dependency management
+tags:
+  - dependencies
+  - dbt
+  - extensibility
+---
+
+Dependencies allow you to make use of third-party or custom code in your Models and Jobs. You can connect dependencies to your SQL projects, and create them from the DBT Hub, GitHub, or another Prophecy Project.
+
+## View dependencies
+
+To see all of your Project and Model dependencies simply open the Manage Dependencies screen by clicking **... > Dependencies**.
+
+![View your dependencies](img/dependencies-view.png)
+
+You can also view your dependencies and get the same Manage Dependencies screen from the bottom left-hand side of the canvas, under **DEPENDENCIES**.
+
+![View your dependencies](img/dependencies-list.png)
+
+## Add dependency
+
+To add the dependency, first navigate to [Manage Dependencies](#view-dependencies). There you will see a complete list of your dependencies. Click on **Add Dependency** to add a new dependency.
+
+You will be taken to the Create Dependency screen, where you can choose to create a dependency from the DBT Hub, GitHub, or another Prophecy Project.
+
+![Add a Dependency](img/dependencies-add.png)
+
+After you've defined the Dependency, click **Create**. Prophecy will validate the dependency and add it to the Project, making it enabled in the current Model, and adding it as a dependency within all of the other Models in the Project.
+
+When adding dependencies, Prophecy validates that the dependency coordinates are valid and accessible. If that fails, you will see an invalid coordinates error.
+
+:::info Invalid coordinates
+
+In rare cases, your dependency might be only accessible to the cluster or the build system but not Prophecy itself. If you're confident that the dependency is correct, but the validation error shows up, it's safe to press **Save Anyways** to ignore that warning.
+
+:::
+
+### Add a DBT dependency
+
+To add a DBT dependency, you must select **DBT Hub** and provide the dbt package and version number.
+
+![Add a DBT Dependency](img/dependencies-dbt.png)
+
+You can find the dbt package and version number from the [dbt Package hub](https://hub.getdbt.com/dbt-labs/dbt_utils/latest/).
+
+![dbt_utils dependency](img/dependencies-dbt-utils.png)
+
+You can copy and paste the information into the Create Dependency fields.
+
+### Add a GitHub dependency
+
+To add a GitHub dependency, you must select **GitHub** and provide the Git Repository, Revision, and Sub Directory.
+
+![Add a GitHub Dependency](img/dependencies-github.png)
+
+The Revision must be either a Git tag, commit has, or branch name.
```

Comment:
@bandikishores on the GitHub Create Dependency menu, do you know what the checkbox "Warn unpinned" does if the user decides to use it?

Reply:
Hey @alexanderahn , this means that users want to point to their GitHub repo without specifying any version/branch/commit. This means we do a copy of the code from GitHub with whatever was the default branch + the latest code. So every time the user runs the project, it is possible we pull a different code (as the remote GitHub we're referring to might have changed and we said we always want the latest) resulting in unexpected behavior for the user on each run. So it's recommended not to enable it.

Reply:
Ack.